tlsr9/crypto/mbedtls/internal/ecp_alt_tlx_backend.c

```diff
@@ -47,8 +47,13 @@
 #include <multithread.h>
 #include <pke.h>
 
-
+#if CONFIG_SOC_RISCV_TELINK_B92
+#include <ext_driver/driver_lib/ext_lib.h>
+#elif CONFIG_SOC_RISCV_TELINK_B95
 #include <ext_driver/driver_lib/ext_lib.h>
+#elif CONFIG_SOC_RISCV_TELINK_TL321X
+#include <ext_driver/driver_internal/ext_lib.h>
+#endif
```

Comment:
What about B91?
maybe better to use next code:
```
#if CONFIG_SOC_RISCV_TELINK_TL321X
#include <ext_driver/driver_internal/ext_lib.h>
#else
#include <ext_driver/driver_lib/ext_lib.h>
#endif
```

Reply:
But this file if TLX and I assume it used only for TLX SoC?
Why we mention here B92, B95?

Reply:
Will leave TL321X only.